gpytorch/variational/whitened_variational_strategy.py

```diff
@@ -189,8 +189,9 @@ def forward(self, x):
             if self.training:
                 data_covariance = DiagLazyTensor((data_data_covar.diag() - interp_data_data_var).clamp(0, math.inf))
             else:
-                neg_induc_data_data_covar = induc_induc_covar.inv_matmul(
-                    induc_data_covar, left_tensor=induc_data_covar.transpose(-1, -2).mul(-1)
+                neg_induc_data_data_covar = torch.matmul(
```

Comment:
Not sure I fully understand the `left_tensor` arg of inv_matmul - is this just for efficiency reasons in the context of CG/preconditioning?